motoko/basic_bitcoin/README.md

```diff
@@ -79,17 +88,48 @@ In the output above, to see the Candid Web UI for your bitcoin canister, you wou
 
 ## Step 2: Generating a Bitcoin address
 
-Bitcoin has different types of addresses (e.g. P2PKH, P2SH). Most of these
-addresses can be generated from an ECDSA public key. The example code
-showcases how your canister can generate a [P2PKH address](https://en.bitcoin.it/wiki/Transaction#Pay-to-PubkeyHash) using the [ecdsa_public_key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-ecdsa_public_key) API.
-
-On the Candid UI of your canister, click the "Call" button under `get_p2pkh_address` to
-generate a P2PKH Bitcoin address.
+Bitcoin has different types of addresses (e.g. P2PKH, P2SH, P2TR). You may want
+to check [this
+article](https://bitcoinmagazine.com/technical/bitcoin-address-types-compared-p2pkh-p2sh-p2wpkh-and-more)
+if you are interested in a high-level comparison of different address types.
+These addresses can be generated from an ECDSA public key or a Schnorr
+([BIP340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki))
+public key. The example code showcases how your canister can generate and spend
+from two types of addresses:
+1. A [P2PKH address](https://en.bitcoin.it/wiki/Transaction#Pay-to-PubkeyHash)
+   using the
+   [ecdsa_public_key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-method-ecdsa_public_key)
+   API.
+2. A [P2TR
+   address](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki)
+   where the funds can be spent using the raw (untweaked) internal key
+   (so-called P2TR key path spend, but untweaked). The advantage of this
+   approach compared to the one below is its significantly smaller fee per
```

Comment:
It's not fully clear to me what "the one below" means here.

Reply:
That's a copy paste. The one below should refer to script spends, which are not yet implemented in this PR. Replaced with "P2TR script spends".